### PR TITLE
Resolve Sweep orchestration logic

### PR DESCRIPTION
**Pull request by tchaton**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning-hpo/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?

Fixes # (issue).

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -25,6 +25,9 @@
 app = LightningApp(
     TrainingStudio(),
     info=frontend.AppInfo(
-        title="Lightning PyTorch Training Studio", description=description  # , on_after_connect=on_after_connect
+        title="Lightning PyTorch Training Studio",
+        description=description,
+        on_connect_end=on_connect_end,
     ),
+    log_level="info",
```
Comment (lantiga) [inline app.py:32]:
```suggestion
```

File: lightning_training_studio/app/ui/src/components/ExperimentTable.tsx [hunk 2]
```diff
@@ -118,10 +124,26 @@ function toProgress(experiment: ExperimentConfig) {
...
+}
+
+function timeLeft(experiment: ExperimentConfig) {
+  if (experiment.end_time || experiment.stage == 'stopped' || experiment.stage == 'failed') {
+    return '';
+  }
+  if (experiment.progress) {
+    const nowTime = getTime();
+    const estimatedEnd =
+      Number(experiment.start_time) + (100 * (nowTime - Number(experiment.start_time))) / experiment.progress;
```
Comment (lantiga) [inline lightning_training_studio/app/ui/src/components/ExperimentTable.tsx:144]: handle when progress is still 0 (and potentially elapsed time is also very small)?

File: lightning_training_studio/components/sweep.py [hunk 3]
```diff
@@ -300,6 +299,19 @@ def _get_objective(self, experiment_id: int):
             )
             setattr(self, f"w_{experiment_id}", objective)
             self.experiments[experiment_id]["stage"] = Stage.PENDING
+
+            # TODO: Remove when display name is merged
+            if hasattr(LightningWork, "_display_name"):
```
Comment (lantiga) [inline lightning_training_studio/components/sweep.py:304]: where do we set `_display_name` as a class attribute of `LightningWork`?

File: lightning_training_studio/app/ui/src/components/ExperimentTable.tsx (hunk 2 above)
Comment (tchaton) [reply lightning_training_studio/app/ui/src/components/ExperimentTable.tsx:144]: It won't happen. progress goes from null to defined.

File: lightning_training_studio/components/sweep.py (hunk 3 above)
Comment (tchaton) [reply lightning_training_studio/components/sweep.py:304]: Right now,

work.display_name = '...'

If everything works fine, we can add it as an class argument.

**Pull request merged**